PreludeTests/ComparatorTests.swift

```diff
@@ -16,7 +16,7 @@ final class ComparatorTests: XCTestCase {
 }
 
 extension Int {
-  static var comparator: Comparator<Int> {
+  static var comparator: Prelude.Comparator<Int> {
```

Comment:
Yeah, I think just because `NSComparator` is now `Comparator`. From Foundation:

```
public typealias Comparator = (Any, Any) -> ComparisonResult
```